api/all.yaml

```diff
@@ -1398,7 +1401,111 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/ExternalLocationInfo"
-
+  /delta/preview/commits:
+    get:
+      operationId: getCommits
+      tags:
+        - CoordinatedCommits
+      summary: |
+        List unbackfilled Delta table commits.
+        WARNING: This API is experimental and may change in future versions.
+      description: |
+        List all the unbackfilled delta commits that are currently being tracked by the UC coordinator. 
+        If no commits are being tracked in the specific version range (from start_version to an optional end_version), 
+        it will return an empty list.
+        WARNING: This API is experimental and may change in future versions.
+      parameters:
+        - name: table_id
+          in: query
+          description: The table ID provided by UC on table creation that we want to get the commits for. In the context of UC, this is the UUID that uniquely identifies a table.
+          required: true
+          schema:
+            type: string
+        - name: table_uri
+          in: query
+          description: |
+            The URI of the storage location of the table. If the table_id exists but the table_uri is 
+            different from the one previously registered (e.g., if the client moved the table), the request will fail.
+          required: true
+          schema:
+            type: string
+        - name: start_version
```

Comment:
Is this `start_version` an auto incremental field, or will it be comparable to the `end_version` ?

Reply:
see comment below. 50 is a implementation detail